packages/help-center/src/components/_mixins.scss

```
@@ -3,96 +3,103 @@
 @import "variables";
 
 @mixin help-center-link {
-	> ul {
-		list-style: none;
-		padding: 0;
-		text-align: left;
-		border: 1px solid var(--studio-gray-5);
-		border-radius: 8px;
-		margin: 0 0 16px;
-		overflow: hidden;
-	}
+    > ul {
+        list-style: none;
+        padding: 0;
+        text-align: left;
+        border: 1px solid var(--studio-gray-5);
+        border-radius: 8px;
+        margin: 0 0 16px;
+        overflow: hidden;
+    }
 
-	.help-center-link__item {
-		line-height: 1.3;
-		border-bottom: 1px solid var(--studio-gray-5);
-		display: block;
+    .help-center-link__item {
+        
+        border-bottom: 1px solid var(--studio-gray-5);
+        display: block;
+        line-height: 1.3;
+        margin-bottom: 0;
 
-		&:first-child {
-			a:focus, button:focus {
-				border-radius: 8px 8px 0 0;
-			}
-		}
+        &:first-child {
+            a:focus, button:focus {
+                border-radius: 8px 8px 0 0;
+            }
+        }
 
-		&:last-child {
-			border-bottom: 0;
+        &:last-child {
+            border-bottom: 0;
 
-			a:focus, button:focus {
-				border-radius: 0 0 8px 8px;
-			}
-		}
+            a:focus, button:focus {
+                border-radius: 0 0 8px 8px;
+            }
+        }
 
-		.help-center-link__cell {
-			display: block;
-			width: 100%;
+        .help-center-link__cell {
+            display: block;
+            width: 100%;
 
-			&:hover {
-				a, button {
-					background: color-mix(in srgb, #fff 98%, var(--wp-admin-theme-color, $help-center-blue));
-					color: var(--wp-components-color-accent, var(--wp-admin-theme-color, $help-center-blue));
-					cursor: pointer;
-				}
+            &:hover {
+                a, button {
+                    background: color-mix(in srgb, #fff 98%, var(--wp-admin-theme-color, $help-center-blue));
+                    color: var(--wp-components-color-accent, var(--wp-admin-theme-color, $help-center-blue));
+                    cursor: pointer;
+                }
 
-				svg {
-					fill: var(--wp-components-color-accent, var(--wp-admin-theme-color, $help-center-blue));
-				}
-			}
+                svg {
+                    fill: var(--wp-components-color-accent, var(--wp-admin-theme-color, $help-center-blue));
+                }
+            }
 
-			a,
-			button {
-				padding-top: 16px;
-				padding-bottom: 16px;
-				display: flex;
-				text-decoration: none;
-				color: var(--studio-gray-100);
-				font-size: $font-body-small;
-				border-radius: 2px;
-				font-weight: normal;
-				align-items: center;
-				transition: background-color 0.3s ease;
-				line-height: 1.4;
-				width: 100%;
-				min-width: 0;
-				box-sizing: border-box;
+            a,
+            button {
+                background-color: transparent;
+                border-width: 0;
+                padding-top: 16px;
+                padding-bottom: 16px;
+                padding-inline-start: 0;
+                padding-inline-end: 0;
```

` all: unset;` should do the job and be more future proof. 

```suggestion
button {
                all: unset;
                padding-top: 16px;
                padding-bottom: 16px;
```